app/views/govuk_publishing_components/components/_input.html.erb

```diff
@@ -1,32 +1,52 @@
 <%
   id ||= "input-#{SecureRandom.hex(4)}"
-  hint_id ||= "hint-#{SecureRandom.hex(4)}"
   value ||= nil
-  error_message ||= false
-  label ||= {}
   type ||= "text"
   describedby ||= false
-  ariadescribedby ||= nil
```

Comment:
Is this a breaking change?

Reply:
no, `ariadescribedby` was introduced to a few commit ago (29ef8a8) to concatenate the ids of the items than need to be read out by assistive tech.
At that point it should've been `ariadescribedby = nil` without `||`, so no breaking change. Now we use `aria_described_by` for that purpose, to be consistent with the other variables names.

Reply:
Sorry, I meant is any app using `ariadescribedby` instead of `aria_described_by`? Would be good to accept the old param with a deprecation warning if so.

Reply:
👍